frontend/server/src/main/java/org/pytorch/serve/wlm/WorkerThread.java

```diff
@@ -447,7 +446,12 @@ public void channelRead0(ChannelHandlerContext ctx, ModelWorkerResponse msg) {
         public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) {
             logger.error("Unknown exception", cause);
             if (cause instanceof OutOfMemoryError) {
-                NettyUtils.sendError(ctx, HttpResponseStatus.INSUFFICIENT_STORAGE, cause);
+                ModelWorkerResponse msg = new ModelWorkerResponse();
+                msg.setCode(HttpURLConnection.HTTP_ENTITY_TOO_LARGE);
```

Comment:
Same as above, can you explain whats the reason for change 507 to 513?

Reply:
HttpURLConnection doesn't support 507 code.